corehq/warehouse/models/facts.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+from corehq.warehouse.const import (
+    APP_STATUS_FACT_SLUG,
+    USER_DIM_SLUG,
+    FORM_STAGING_SLUG,
+    SYNCLOG_STAGING_SLUG,
+)
+
+from .dimensions import UserDim
+from .shared import WarehouseTableMixin
+
+
+class ApplicationStatusFact(models.Model, WarehouseTableMixin):
+    '''
+    Application Status Report Fact Table
+
+    Grain: app_id, user_id
```

Comment:
great to see this! we should include the grain in our docs for all facts and dims